apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/extract/RifFilesProcessorTest.java

```diff
@@ -659,7 +759,102 @@ public void process1SNFClaimRecord() {
+    assertEquals("4444444444", claimGroup.getOtherPhysicianNpi().get());
+    assertEquals("1", claimGroup.getPatientDischargeStatusCode());
+    assertEquals(new BigDecimal("5555.03"), claimGroup.getTotalChargeAmount());
+    assertEquals(LocalDate.of(2013, 11, 5), claimGroup.getClaimAdmissionDate().get());
+    assertEquals('3', claimGroup.getAdmissionTypeCd());
+    assertEquals('4', claimGroup.getSourceAdmissionCd().get().charValue());
+    assertEquals(new BigDecimal("112.00"), claimGroup.getDeductibleAmount());
+    assertEquals(new BigDecimal("5.00"), claimGroup.getPartACoinsuranceLiabilityAmount());
+    assertEquals(new BigDecimal("6.00"), claimGroup.getBloodDeductibleLiabilityAmount());
+    assertEquals(new BigDecimal("33.00"), claimGroup.getNoncoveredCharge());
+    assertEquals(new BigDecimal("14.00"), claimGroup.getTotalDeductionAmount());
+    assertEquals(new BigDecimal("9.00"), claimGroup.getClaimPPSCapitalFSPAmount().get());
+    assertEquals(new BigDecimal("8.00"), claimGroup.getClaimPPSCapitalOutlierAmount().get());
+    assertEquals(
+        new BigDecimal("7.00"), claimGroup.getClaimPPSCapitalDisproportionateShareAmt().get());
+    assertEquals(new BigDecimal("6.00"), claimGroup.getClaimPPSCapitalIMEAmount().get());
+    assertEquals(new BigDecimal("5.00"), claimGroup.getClaimPPSCapitalExceptionAmount().get());
+    assertEquals(
+        new BigDecimal("4.00"), claimGroup.getClaimPPSOldCapitalHoldHarmlessAmount().get());
+    assertEquals(BigDecimal.valueOf(17L), claimGroup.getUtilizationDayCount());
+    assertEquals(BigDecimal.valueOf(17L), claimGroup.getCoinsuranceDayCount());
+    assertEquals(BigDecimal.ZERO, claimGroup.getNonUtilizationDayCount());
+    assertEquals(BigDecimal.valueOf(19L), claimGroup.getBloodPintsFurnishedQty());
+    assertEquals(LocalDate.of(2013, 9, 23), claimGroup.getQualifiedStayFromDate().get());
+    assertEquals(LocalDate.of(2013, 11, 5), claimGroup.getQualifiedStayThroughDate().get());
+    assertEquals(LocalDate.of(2002, 1, 11), claimGroup.getNoncoveredStayFromDate().get());
+    assertEquals(LocalDate.of(2002, 1, 21), claimGroup.getNoncoveredStayThroughDate().get());
+    assertFalse(claimGroup.getCoveredCareThroughDate().isPresent());
+    assertEquals(LocalDate.of(2002, 1, 31), claimGroup.getMedicareBenefitsExhaustedDate().get());
+    assertEquals(LocalDate.of(2013, 12, 18), claimGroup.getBeneficiaryDischargeDate().get());
+
+    assertEquals("6202", claimGroup.getDiagnosisAdmittingCode().get());
+    assertEquals('9', claimGroup.getDiagnosisAdmittingCodeVersion().get().charValue());
+    assertEquals("E9281", claimGroup.getDiagnosisExternal1Code().get());
+    assertEquals('9', claimGroup.getDiagnosisExternal1CodeVersion().get().charValue());
+    assertEquals("23443453453", claimGroup.getFiDocumentClaimControlNumber().get());
+    assertEquals("34534535535", claimGroup.getFiOriginalClaimControlNumber().get());
+    assertEquals("6455", claimGroup.getDiagnosisRelatedGroupCd().get());
```

Comment:
Maybe throw a comment here that this is the field we're primarily testing, just cause it's hard to find in the noise